cmd/copy.go

```diff
@@ -1542,6 +1542,17 @@ func (cca *CookedCopyCmdArgs) processCopyJobPartOrders() (err error) {
 	case cca.FromTo.IsDelete():
 		// Delete gets ran through copy, so handle delete
 		if cca.FromTo.From() == common.ELocation.BlobFS() {
+			if cca.Recursive { // Extra hacky hack: Carrying recursive into STE temporarily to enable cancel of delete job
```

Comment:
is this so that we don't have to add an extra entry to the plan file?

Reply:
Should we just add it to the plan file so it's cleaner? This change seems not friendly to those that may not have historical knowledge in the future.

Reply:
> Should we just add it to the plan file so it's cleaner? This change seems not friendly to those that may not have historical knowledge in the future.

Probably.